gcloud/bigtable/happybase/table.py

```diff
@@ -561,3 +565,135 @@ def _gc_rule_to_dict(gc_rule):
                 if key1 != key2:
                     result = {key1: rule1[key1], key2: rule2[key2]}
     return result
+
+
+def _next_char(str_val, index):
+    """Gets the next character based on a position in a string.
+
+    :type str_val: str
+    :param str_val: A string containing the character to update.
+
+    :type index: int
+    :param index: An integer index in ``str_val``.
+
+    :rtype: str
+    :returns: The next character after the character at ``index``
+              in ``str_val``.
+    """
+    if six.PY3:  # pragma: NO COVER
```

Comment:
Thanks for persisting on this. I should've have read the `six` docs closer. The fact that

```python
>>> # PY2
>>> (b'ABC')[0]
'a'
>>> # PY3
>>> (b'ABC')[0]
65
```

is what makes this difficult but `six.indexbytes(b'ABC', 0)` produces 65 every time.